### PR TITLE
OPRUN-3106: skuznets/bump vendor ii

### PR DESCRIPTION
**Pull request by stevekuznetsov**

### CODE REVIEW

**Comment by awgreene** [comment]
/approve
/lgtm

There may be a need to add https://github.com/openshift/operator-framework-olm/pull/601 to this so OLM can use the correct SCC.

**Comment by awgreene** [comment]
Infra failure on ci/prow/e2e-upgrade:
```
level=error
level=error msg=Error: creating EC2 Instance: InvalidNetworkInterfaceID.NotFound: The networkInterface ID 'eni-008fc950d67416d0b' does not exist
level=error msg=	status code: 400, request id: 555737c5-58c4-4a82-a15f-f67a9510a774
level=error
level=error msg=  with module.masters.aws_instance.master[1], 
```

/retest

File: manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ rules:
       - securitycontextconstraints
     resourceNames:
       - restricted-v2
+      - anyuid
```
Comment (awgreene) [inline manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml:31]: This is needed to satisfy a feature in which we allow catalogSources in legacy mode, which cannot run under the constraints introduced by the `restricted-v2` SCC, you can learn more by inspecting [this test](https://github.com/openshift/operator-framework-olm/blob/2ff0c6731160ed93ae9d356e1d1c50e2bafaa06a/staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go#L1540-L1612).

**Comment by stevekuznetsov** [comment]
```
 Starting CatalogSource e2e tests configure gRPC registry pod to extract content 
Message:"failed to populate resolver cache from source catalog-7t7zb/openshift-catsrc-e2e-xqmg6: failed to list bundles: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.224.120:50051: connect: connection refused\""
```

Looks bona-fide here

**Comment by stevekuznetsov** [comment]
Nothing obvious in the must-gather:

`catalog-operator`:

```
2023-11-03T19:34:35.058377185Z time="2023-11-03T19:34:35Z" level=error msg="UpdateStatus - error while setting CatalogSource status" error="Operation cannot be fulfilled on catalogsources.operators.coreos.com \"catalog-lp5hw\": the object has been modified; please apply your changes to the latest version and try again" id=+zQfF source=catalog-lp5hw
2023-11-03T19:34:35.058510525Z E1103 19:34:35.058490       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: Operation cannot be fulfilled on catalogsources.operators.coreos.com "catalog-lp5hw": the object has been modified; please apply your changes to the latest version and try again
2023-11-03T19:34:56.088379536Z time="2023-11-03T19:34:56Z" level=error msg="UpdateStatus - error while setting CatalogSource status" error="Operation cannot be fulfilled on catalogsources.operators.coreos.com \"catalog-lp5hw\": the object has been modified; please apply your changes to the latest version and try again" id=s7oOB source=catalog-lp5hw
2023-11-03T19:34:56.088379536Z E1103 19:34:56.088264       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: Operation cannot be fulfilled on catalogsources.operators.coreos.com "catalog-lp5hw": the object has been modified; please apply your changes to the latest version and try again
2023-11-03T19:34:56.110049896Z time="2023-11-03T19:34:56Z" level=error msg="UpdateStatus - error while setting CatalogSource status" error="Operation cannot be fulfilled on catalogsources.operators.coreos.com \"catalog-lp5hw\": the object has been modified; please apply your changes to the latest version and try again" id=/lWgz source=catalog-lp5hw
2023-11-03T19:34:56.110049896Z E1103 19:34:56.109994       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: Operation cannot be fulfilled on catalogsources.operators.coreos.com "catalog-lp5hw": the object has been modified; please apply your changes to the latest version and try again
2023-11-03T19:35:04.438011155Z time="2023-11-03T19:35:04Z" level=error msg="UpdateStatus - error while setting CatalogSource status" error="Operation cannot be fulfilled on catalogsources.operators.coreos.com \"catalog-lp5hw\": the object has been modified; please apply your changes to the latest version and try again" id=lLBye source=catalog-lp5hw
2023-11-03T19:35:04.438141765Z E1103 19:35:04.438121       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: Operation cannot be fulfilled on catalogsources.operators.coreos.com "catalog-lp5hw": the object has been modified; please apply your changes to the latest version and try again
2023-11-03T19:35:04.889149213Z E1103 19:35:04.888921       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:04.892476612Z E1103 19:35:04.892120       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:04.898688642Z E1103 19:35:04.898629       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:04.923059502Z E1103 19:35:04.923004       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:04.967618532Z E1103 19:35:04.967491       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:05.051932332Z E1103 19:35:05.051875       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:05.089286521Z E1103 19:35:05.089213       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:05.216213951Z E1103 19:35:05.216099       1 queueinformer_operator.go:319] sync {"update" "openshift-catsrc-e2e-7fcjk/catalog-lp5hw"} failed: couldn't ensure registry server - error ensuring service account: catalog-lp5hw: serviceaccounts "catalog-lp5hw" is forbidden: unable to create new content in namespace openshift-catsrc-e2e-7fcjk because it is being terminated
2023-11-03T19:35:05.310442600Z time="2023-11-03T19:35:05Z" level=info msg="removed client for deleted catalogsource" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
```

`packageserver`:
```
2023-11-03T19:34:34.851857826Z time="2023-11-03T19:34:34Z" level=info msg="connecting to source" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:34.856937457Z W1103 19:34:34.856832       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused". Reconnecting...
2023-11-03T19:34:35.247542264Z time="2023-11-03T19:34:35Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:35.247601514Z time="2023-11-03T19:34:35Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:35.444364943Z time="2023-11-03T19:34:35Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:35.444530243Z time="2023-11-03T19:34:35Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:36.552236966Z time="2023-11-03T19:34:36Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:55.858830878Z W1103 19:34:55.858746       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout". Reconnecting...
2023-11-03T19:34:55.859024788Z time="2023-11-03T19:34:55Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:55.859075098Z time="2023-11-03T19:34:55Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:55.859182058Z time="2023-11-03T19:34:55Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:56.079152846Z time="2023-11-03T19:34:56Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:56.079358286Z time="2023-11-03T19:34:56Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:56.100509116Z time="2023-11-03T19:34:56Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:56.100574566Z time="2023-11-03T19:34:56Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:35:04.429239375Z time="2023-11-03T19:35:04Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:35:05.310421820Z time="2023-11-03T19:35:05Z" level=warning msg="error getting bundle data" action="refresh cache" err="rpc error: code = Canceled desc = grpc: the client connection is closing" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:35:05.310829210Z time="2023-11-03T19:35:05Z" level=warning msg="error getting package stream" action="refresh cache" err="rpc error: code = Canceled desc = grpc: the client connection is closing" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:34.851907135Z time="2023-11-03T19:34:34Z" level=info msg="connecting to source" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:34.855663674Z W1103 19:34:34.855588       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused". Reconnecting...
2023-11-03T19:34:35.247718081Z time="2023-11-03T19:34:35Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:35.247892101Z time="2023-11-03T19:34:35Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:35.444474815Z time="2023-11-03T19:34:35Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:35.444474815Z time="2023-11-03T19:34:35Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: connect: connection refused\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:36.552021222Z time="2023-11-03T19:34:36Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:55.856364131Z W1103 19:34:55.856292       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout". Reconnecting...
2023-11-03T19:34:55.856418301Z time="2023-11-03T19:34:55Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:55.856432181Z time="2023-11-03T19:34:55Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:55.856513321Z time="2023-11-03T19:34:55Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:56.079419992Z time="2023-11-03T19:34:56Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:56.079537862Z time="2023-11-03T19:34:56Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:34:56.100610480Z time="2023-11-03T19:34:56Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:34:56.100666180Z time="2023-11-03T19:34:56Z" level=warning msg="error getting bundle stream" action="refresh cache" err="rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.104.104:50051: i/o timeout\"" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:35:04.429113421Z time="2023-11-03T19:35:04Z" level=info msg="updating PackageManifest based on CatalogSource changes: {catalog-lp5hw openshift-catsrc-e2e-7fcjk}" action="sync catalogsource" address="catalog-lp5hw.openshift-catsrc-e2e-7fcjk.svc:50051" name=catalog-lp5hw namespace=openshift-catsrc-e2e-7fcjk
2023-11-03T19:35:05.310717216Z time="2023-11-03T19:35:05Z" level=warning msg="error getting bundle data" action="refresh cache" err="rpc error: code = Canceled desc = grpc: the client connection is closing" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
2023-11-03T19:35:05.310817886Z time="2023-11-03T19:35:05Z" level=warning msg="error getting package stream" action="refresh cache" err="rpc error: code = Canceled desc = grpc: the client connection is closing" source="{catalog-lp5hw openshift-catsrc-e2e-7fcjk}"
```

**Comment by stevekuznetsov** [comment]
Actually looks like the test in question is ` Starting CatalogSource e2e tests image update ` ... which has entirely different semantics locally vs in OCP e2e downstream, so not super surprised it fails here.

There is also no logging in there whatsoever - adding it in https://github.com/operator-framework/operator-lifecycle-manager/pull/3089, will rebase once that's merged upstream to pull it in.

**Comment by stevekuznetsov** [comment]
```
 Starting CatalogSource e2e tests image update
/go/src/github.com/openshift/operator-framework-olm/staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go:805
  created the openshift-catsrc-e2e-4rdwp testing namespace
  STEP: Create an image based catalog source from public Quay image using a unique tag as identifier @ 11/05/23 23:24:40.535
  STEP: Using the OpenShift registry at image-registry.openshift-image-registry.svc:5000/openshift-catsrc-e2e-4rdwp @ 11/05/23 23:24:40.535
  STEP: Generating a target test image name docker://image-registry.openshift-image-registry.svc:5000/openshift-catsrc-e2e-4rdwp/catsrc-update: with tag xrl7dc @ 11/05/23 23:24:40.737
  STEP: copying old catalog image into test-specific tag in internal docker registry @ 11/05/23 23:24:40.738
  STEP: creating a skopoeo Pod to do the copying @ 11/05/23 23:24:40.738
  STEP: waiting for the skopeo pod to exit successfully @ 11/05/23 23:24:40.835
  STEP: removing the skopeo pod @ 11/05/23 23:24:59.881
  STEP: setting catalog source @ 11/05/23 23:25:00.088
  STEP: Create gRPC CatalogSource using an external registry image and poll interval @ 11/05/23 23:25:00.088
  STEP: Wait for a new registry pod to be created @ 11/05/23 23:25:00.129
  Waiting for pods matching selector olm.catalogSource=catalog-sr7nm to match given conditions
  STEP: Create a Subscription for package @ 11/05/23 23:25:00.278
  STEP: Wait for the Subscription to succeed @ 11/05/23 23:25:00.319
  23:30:00.5276: subscription openshift-catsrc-e2e-4rdwp/sub-m64t2 never got correct status: v1alpha1.SubscriptionStatus{CurrentCSV:"", InstalledCSV:"", Install:(*v1alpha1.InstallPlanReference)(nil), State:"", Reason:"", InstallPlanGeneration:0, InstallPlanRef:(*v1.ObjectReference)(nil), CatalogHealth:[]v1alpha1.SubscriptionCatalogHealth{v1alpha1.SubscriptionCatalogHealth{CatalogSourceRef:(*v1.ObjectReference)(0xc0008d97a0), LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local), Healthy:true}, v1alpha1.SubscriptionCatalogHealth{CatalogSourceRef:(*v1.ObjectReference)(0xc0008d9880), LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local), Healthy:true}, v1alpha1.SubscriptionCatalogHealth{CatalogSourceRef:(*v1.ObjectReference)(0xc0008d98f0), LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local), Healthy:true}, v1alpha1.SubscriptionCatalogHealth{CatalogSourceRef:(*v1.ObjectReference)(0xc0008d9960), LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local), Healthy:true}, v1alpha1.SubscriptionCatalogHealth{CatalogSourceRef:(*v1.ObjectReference)(0xc0008d99d0), LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local), Healthy:true}}, Conditions:[]v1alpha1.SubscriptionCondition{v1alpha1.SubscriptionCondition{Type:"CatalogSourcesUnhealthy", Status:"False", Reason:"AllCatalogSourcesHealthy", Message:"all available catalogsources are healthy", LastHeartbeatTime:<nil>, LastTransitionTime:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local)}, v1alpha1.SubscriptionCondition{Type:"ResolutionFailed", Status:"True", Reason:"ErrorPreventedResolution", Message:"failed to populate resolver cache from source catalog-sr7nm/openshift-catsrc-e2e-4rdwp: failed to list bundles: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing dial tcp 172.30.231.90:50051: connect: connection refused\"", LastHeartbeatTime:<nil>, LastTransitionTime:<nil>}}, LastUpdated:time.Date(2023, time.November, 5, 23, 25, 0, 0, time.Local)}
  23:30:00.5277: subscription openshift-catsrc-e2e-4rdwp/sub-m64t2 spec: &v1alpha1.SubscriptionSpec{CatalogSource:"catalog-sr7nm", CatalogSourceNamespace:"openshift-catsrc-e2e-4rdwp", Package:"busybox", Channel:"alpha", StartingCSV:"", InstallPlanApproval:"Automatic", Config:(*v1alpha1.SubscriptionConfig)(nil)}
  [FAILED] in [It] - /go/src/github.com/openshift/operator-framework-olm/staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go:933 @ 11/05/23 23:30:00.935
  collecting the openshift-catsrc-e2e-4rdwp namespace artifacts as the 'image update' test case failed 
```

sigh

**Comment by awgreene** [comment]
>End-to-end: [It] Starting CatalogSource e2e tests image update expand_less

This should have passed, spinning up a cluster to check it out.

**Comment by tmshort** [comment]
>Starting CatalogSource e2e tests image update
>/go/src/github.com/openshift/operator-framework-olm/staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go:806

Yup, that's what I saw.

**Comment by awgreene** [comment]
```
lusteroperator/kube-apiserver is not available (<missing>) because <missing>
	clusteroperator/kube-controller-manager is not available (<missing>) because <missing>
	clusteroperator/kube-scheduler is not available (<missing>) because <missing>
	clusteroperator/kube-storage-version-migrator is not available (<missing>) because <missing>
	clusteroperator/machine-api is not available (<missing>) because <missing>
	clusteroperator/machine-approver is not available (<missing>) because <missing>
	clusteroperator/machine-config is not available (<missing>) because <missing>
	clusteroperator/marketplace is not available (<missing>) because <missing>
	clusteroperator/monitoring is not available (<missing>) because <missing>
	clusteroperator/network is not available (<missing>) because <missing>
	clusteroperator/node-tuning is not available (<missing>) because <missing>
	clusteroperator/openshift-apiserver is not available (<missing>) because <missing>
	clusteroperator/openshift-controller-manager is not available (<missing>) because <missing>
	clusteroperator/openshift-samples is not available (<missing>) because <missing>
	clusteroperator/operator-lifecycle-manager is not available (<missing>) because <missing>
	clusteroperator/operator-lifecycle-manager-catalog is not available (<missing>) because <missing>
	clusteroperator/operator-lifecycle-manager-packageserver is not available (<missing>) because <missing>
	clusteroperator/service-ca is not available (<missing>) because <missing>
	clusteroperator/storage is not available (<missing>) because <missing>
```

/retest

**Comment by awgreene** [comment]
/retest

Flakies gonna flake

**Comment by stevekuznetsov** [comment]
@awgreene not sure the bots will be happy trying to make more than one Jira connected

**Comment by awgreene** [comment]
>@awgreene not sure the bots will be happy trying to make more than one Jira connected

@stevekuznetsov this is a supported workflow, Alex Pavel had sent an email out a few months ago:
>... The Jira GitHub integration has recently been updated to support linking multiple bugs from OCPBUGS to a single PR. To link multiple bugs to a PR, simply include all bugs as a comma-separated list in the title (e.g. “OCPBUGS-123,OCPBUGS-124: Fixed the issue”). Support for multiple bugs should work for all the usual PR events that the integration usually handles, including cherrypicks and the `/jira cherrypick` command...

**Comment by awgreene** [comment]
/retitle OCPBUGS-21980,OCPBUGS-20347,OPRUN-3106: skuznets/bump vendor ii

**Comment by grokspawn** [comment]
/retitle OPECO-3054,OCPBUGS-21980,OCPBUGS-20347,OPRUN-3106: skuznets/bump vendor ii

**Pull request merged**